src/Microsoft.TemplateEngine.Abstractions/Mount/IMountPoint.cs

```diff
@@ -4,16 +4,37 @@ namespace Microsoft.TemplateEngine.Abstractions.Mount
 {
     public interface IMountPoint : IDisposable
     {
-        string AbsoluteUri { get; }
+        /// <summary>
+        /// Returns mount point URI (as received from source = not normalized).
+        /// </summary>
+        string MountPointUri { get; }
 
+        /// <summary>
+        /// Returns root directory of the mount point.
+        /// </summary>
         IDirectory Root { get; }
 
         IEngineEnvironmentSettings EnvironmentSettings { get; }
 
+        /// <summary>
+        /// Gets the file info for the file in the mount point.
+        /// </summary>
+        /// <param name="fullPath">The path to the file relative to mount point root.</param>
+        /// <returns></returns>
         IFile FileInfo(string fullPath);
```

Comment:
Should we rename this `fullPath` parameters to something mentioning this is full path inside mount point?